asgs_main.sh

```diff
@@ -2172,22 +2177,46 @@ while [ true ]; do
          $ADCIRCDIR/aswip -n $BASENWS >> ${SYSLOG} 2>&1
          if [[ -e NWS_${BASENWS}_fort.22 ]]; then
             mv fort.22 fort.22.orig >> ${SYSLOG} 2>&1
-            cp NWS_${BASENWS}_fort.22 fort.22 >> ${SYSLOG} 2>&1
+            if [[ $BACKGROUNDMET != "namBlend" ]]; then
```

Comment:
I think we need to prefer managing these as `case/esac` just to plan for growth.  Also, checking a _negative_ (e.g., `!= "namBlend")` is asking for trouble, let's make this more positive :) - and `case/esac`.